kuma/wiki/views/document.py

```diff
@@ -799,48 +801,7 @@ def react_document(request, document_slug, document_locale):
                                                         document_slug)
 
     if doc is None:
-        # Possible the document once existed, but is now deleted.
-        # If so, show that it was deleted.
-        deletion_log_entries = DocumentDeletionLog.objects.filter(
-            locale=document_locale,
-            slug=document_slug
-        )
-        if deletion_log_entries.exists():
-            # Show deletion log and restore / purge for soft-deleted docs
-            deleted_doc = Document.deleted_objects.filter(
-                locale=document_locale, slug=document_slug)
-            if deleted_doc.exists():
-                return _document_deleted(request, deletion_log_entries)
-
-        # We can throw a 404 immediately if the request type is HEAD.
-        # TODO: take a shortcut if the document was found?
-        if request.method == 'HEAD':
-            raise Http404
-
-        # Check if we should fall back to default locale.
-        fallback_doc, fallback_reason, redirect_url = _default_locale_fallback(
```

Comment:
Thanks, Ryan. I was taking a weed wacker approach there, and just tossing out stuff wholesale. 
Eventually I think I'll also need to add back in some way of handling content experiments, which also got tossed here.

For the redirect case, I could try to convert /docs/ to /ducks/ in the URL. But I think I won't because that will just break when we switch to a beta subdomain.